test/mock/galera_test.sh

```
@@ -0,0 +1,101 @@
+    create_mock_galera_version $version
+done
+
+function check_sst_method {
+    dir=$1
+    expected=$2
+
+    my_file=$SANDBOX_HOME/$dir/node1/my.sandbox.cnf
+    ok_file_exists $my_file
+    ok "expected is defined" "$expected"
+
+    found=$(grep "wsrep_sst_method\s*=\s*$expected" $my_file )
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)